sway/desktop/render.c

```diff
@@ -1985,8 +1883,7 @@ void output_render(struct sway_output *output, struct timespec *when,
 	render_drag_icons(output, damage, &root->drag_icons);
 
 renderer_end:
-	// Draw the contents of our buffer into the wlr buffer
-	fx_framebuffer_bind(&renderer->wlr_buffer);
+	fx_renderer_end(output->renderer);
 
 	float clear_color[] = {0.0f, 0.0f, 0.0f, 1.0f};
 	if (pixman_region32_not_empty(&extended_damage)) {
```

Comment:
We shouldn't need to clear the wlrs buffer region when we will be drawing over it anyways.

Then we could also move the `render_whole_output` and disabling of the stencil calls below into `fx_renderer_end`